File: BuddhaGameClient/Assets/Scripts/Utils/Log/Outputs/MultipleLogOutput.cs
using System.Collections;
using System.Collections.Generic;
using UnityEngine;

namespace BuddhaGame
{
    public class MultipleLogOutput : LogOutput
    {
        private LogOutput[] mLogOutputs;

        public MultipleLogOutput(params LogOutput[] args)
        {
            mLogOutputs = args;
        }


        public override void write(int level, string message)
        {
            if (mLogOutputs != null)
            {
                foreach (LogOutput output in mLogOutputs)
                {
                    output.write(level, message);
                }
            }
        }
    }
}

File: BuddhaGameServer/src/log/outputs/MultipleLogOutput.java
package log.outputs;

public class MultipleLogOutput extends LogOutput{

    private LogOutput[] mLogOutputs;

    public MultipleLogOutput(LogOutput...args) {
        mLogOutputs = args;
    }

    @Override
    public void write(int level, String message) {
        if(mLogOutputs != null){
            for(LogOutput output : mLogOutputs){
                output.write(level, message);
            }
        }
    }
}
